light-base/examples/basic.rs

```diff
@@ -81,32 +79,70 @@ fn main() {
             user_data: (),
         })
         .unwrap();
-
-    // The chain is now properly initialized.
+    // Connection to Polkadot is now properly initialized.
 
     // `json_rpc_responses` can only be `None` if we had passed `json_rpc: Disabled` in the
     // configuration.
-    let mut json_rpc_responses = json_rpc_responses.unwrap();
+    let mut polkadot_json_rpc_responses = polkadot_connection.json_rpc_responses.unwrap();
 
-    // Send a JSON-RPC request to the chain.
+    // Ask the client to connect to Assethub.
+    let assethub_connection = client
+        .add_chain(smoldot_light::AddChainConfig {
+            specification: include_str!("../../demo-chain-specs/asset-hub-polkadot.json"),
```

Comment:
Please rename to `polkadot-asset-hub.json`. I'm using the relay chain name as a prefix so that it's not too chaotic.